### ISSUE TITLE
dependency on symfony/config

### ISSUE THREAD

**Issue by flip111** [jeromemacias/Silex-Debug#4]
I assume a dependency to symfony/config is created by [this line](https://github.com/jeromemacias/Silex-Debug/blob/master/DebugServiceProvider.php#L68) perhaps another implementation is possible here?? Also composer.json could [suggest](https://getcomposer.org/doc/04-schema.md#suggest) to install symfony/config. This is not an issue of the original symfony debug-bundle because it is an [optional requirement there](https://github.com/symfony/symfony/issues/13614)

**Comment by jeromemacias** [comment]
I don't see any reason for usage of config dependency. I will make a check.

**Comment by flip111** [comment]
Hi, this seems to be related to this line in the code https://github.com/jeromemacias/Silex-Debug/blob/ef05e5dbfeb3a3d8a3bb5c6e5756768f92688075/DebugServiceProvider.php#L68

I was requiring symfony/config manually in my composer.json as workaround but i keep tripping over this each time i start a new project.

I understand that it's difficult to hardcode a path especially with PSR-0 vs PSR-4. However if you want to use reflection then it seems that it's a hard requirement to have the packages that are used in the Configuration class. I wouldn't know any other elegant solution.

I changed the vendor file as temporary workaround to `return __DIR__.'/../../symfony/debug-bundle/Resources/views';`

**Comment by stof** [comment]
@flip111 this would not work in case the templates are installed in a different location (through the `symfony/symfony` package for instance)

**Comment by flip111** [comment]
@stof yes of course ... like i said it's a temporary workaround (that works in my particular case) ... also it's not good to hardcore the path because it can change because of different reasons ...

**Comment by stof** [comment]
@flip111 please try my fix
